include/llbuild/BuildSystem/BuildValue.h

```diff
@@ -52,15 +52,21 @@ class BuildValue {
 
     /// A value for a produced output whose command failed.
     FailedInput,
+      
+    /// A value for a produced output whose command was cancelled.
+    CancelledInput,
```

Comment:
Okay! I didn't notice this. I wonder if we can encode this dependency statically somehow instead of having to remember to bump it?